tests/test_digital.py

```diff
@@ -342,3 +350,65 @@ def test_operator_product(gate: Primitive) -> None:
     assert torch.allclose(
         op_mul, torch.eye(2**n_qubits, dtype=torch.cdouble).unsqueeze(2).repeat(1, 1, max_batch)
     )
+
+
+@pytest.mark.parametrize("FlipGate", [pyq.BitFlip, pyq.PhaseFlip, pyq.Depolarizing])
```

Comment:
```suggestion
@pytest.mark.parametrize("noisy_flip_gates", [BitFlip, PhaseFlip, Depolarizing])
```

And import noise gates.

Reply:
Changed.